data/mt/retired/Kathy-Kelker-2b3be8a4-c4bb-423d-8a66-c9b1669215ff.yml

```diff
@@ -3,18 +3,24 @@ name: Kathy Kelker
 given_name: Kathy
 family_name: Kelker
 gender: Female
-email: Kathy.Kelker@mtleg.gov
+email: Kathy.Kelker@legmt.gov
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-31'
```

Comment:
@bfossen-ce this person was in the house from 2015 until 01/02/2023. On that date they are starting in the senate